project11/project11.sql

```diff
@@ -0,0 +1,248 @@
+
+-- Функции объектов
+CREATE TYPE function AS ENUM ('Жилое помещение', 'Кафе', 'Спорт. площадка', 'Бассейн', 'Штаб делегации', 'Столовая');
```

Comment:
I'd rather call it object_type, but it's more a nitpick.

Reply:
done